docs/developer/reference/device-standards.rst

```diff
@@ -0,0 +1,87 @@
+Ophyd vs Ophyd-async
+====================
+Some devices have been written in ``ophyd`` for historic reasons. However, all new devices should be written in 
+``ophyd-async`` and if old ``ophyd`` devices are undergoing a large re-write they should also be considered for 
+conversion to ``ophyd-async``. 
+
+Where to put devices
+====================
+
+Dodal is written with the philosophy that Ophyd devices should be assumed to be as generic as possible. I.e. you 
+should think about where to place them in the following order:
+
+#. A device that could be used at any facility, e.g. a generic ``EpicsMotor`` or a commercial product with a 
+   standard IOC, should go in https://github.com/bluesky/ophyd-epics-devices
+#. A device that may be on any beamline should go in the top level of the ``devices`` folder. If it is a quite 
+   complex device (e.g. multiple files) it should have a folder of it's own i.e. ``oav``
+#. A device that is very specific to a particular beamline should go in the ``devices/ixx`` folder
+
+This is in an effort to avoid duplication across facilities/beamlines. 
+
+PV Suffixes
+===========
+
+In general devices should contain only the PV suffixes that are generic for any instance of the device e.g.
+
+.. code-block:: python
+
+    class MyDevice(Device):
+        bragg: EpicsMotor = Cpt(EpicsMotor, "BRAGG")
+    
+    device_instantiation(DCM, "dcm", "-MO-DCM-01:")
+
+
+is preferred over
+
+.. code-block:: python
+
+    class MyDevice(Device):
+        bragg: EpicsMotor = Cpt(EpicsMotor, "-MO-DCM-01:BRAGG")
+
+    device_instantiation(DCM, "dcm", "")
+
+This is so that a new device on say ``-MO-DCM-02`` can be easily created.
```

Comment:
Nit(s). 
- Since we're saying we should write new devices in `ophyd_async` maybe we could update the examples to reflect that?

- I think it would also be useful to add a couple of lines on how to deal with PVs that do not use a standard beamline suffix, in which case `device_instantation` should set `bl_prefix=False`.
It's not super common, but something I run into in the past and it would be nice to have a small example for the docs.